Auth/README.md

```diff
@@ -18,7 +18,8 @@ FirebaseUI can be easily customized to fit in with the rest of your app's visual
 style, and it is open source, so you aren't constrained in realizing the user
 experience you want.
 
-Compatible FirebaseUI clients are also available for [Android](https://github.com/firebase/firebaseui-android/tree/master/auth)
+Compatible FirebaseUI Auth clients are also available for
+[Android](https://github.com/firebase/firebaseui-android/tree/master/auth)
```

Comment:
URL for the web doesn't contain the path to the master branch, can we shorten this one here to https://github.com/firebase/firebaseui-android/ ?

Reply:
The reason these two URLs are different is FirebaseUI Android contains non-Auth components but FirebaseUI Web does not.